wallet/font-icons/README.md

```diff
@@ -23,6 +23,8 @@ It will be useful to also document here the source of each icon.
 -   [coins.svg](./svgs/coins.svg) - exported from [figma](https://www.figma.com/file/OzLaRFzevjxdQAbybWEZk0/Sui-Visualize?node-id=2251%3A47447)
 -   [hand-coins.svg](./svgs/hand-coins.svg) - exported from [figma](https://www.figma.com/file/OzLaRFzevjxdQAbybWEZk0/Sui-Visualize?node-id=2251%3A47447)
 -   [percentage-polygon.svg](./svgs/percentage-polygon.svg) - exported from [figma](https://www.figma.com/file/OzLaRFzevjxdQAbybWEZk0/Sui-Visualize?node-id=2251%3A47447)
+-   [close.svg](./svgs/close.svg) - exported from [figma](https://www.figma.com/file/rkFrheddol8YO7HQaHgIfF/Sui-Systematize?node-id=3421%3A2392)
+-   [arrow-right.svg](./svgs/arrow-right.svg) - exported from [figma](https://www.figma.com/file/rkFrheddol8YO7HQaHgIfF/Sui-Systematize?node-id=3421%3A2392)
```

Comment:
wondering whether these should be Figma since referring to a formal product.

Reply:
figma -> Figma ?

or remove link?